e2e/test/scenarios/visualizations-charts/scatter.cy.spec.js

```diff
@@ -132,7 +133,7 @@ function triggerPopoverForBubble(index = 13) {
     cy.findByLabelText("Switch to visualization").click(); // ... and then back to the scatter visualization (that now seems to be stable enough to make assertions about)
   });
 
-  cy.get(".bubble")
+  lineChartCircle()
```

Comment:
Nit: maybe we can rename it to `cartesianChartCircle` so it makes sense for both line and scatter charts